projects/ag-grid/documentation/code-examples/data-grid/basic/basic-data-grid-docs-demo.component.ts

```diff
@@ -0,0 +1,119 @@
+import {
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  Component,
+} from '@angular/core';
+import { SkyCellType, SkyAgGridService } from '@skyux/ag-grid';
+
+import {
+  GridApi,
+  GridReadyEvent,
+  GridOptions,
+  ValueFormatterParams,
+} from 'ag-grid-community';
+
+import { SKY_AG_GRID_DEMO_DATA } from './basic-data-grid-docs-demo-data';
+
+@Component({
+  selector: 'app-basic-data-grid-docs-demo',
+  templateUrl: './basic-data-grid-docs-demo.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class SkyBasicDataGridDemoComponent {
+  public columnDefs = [
+    {
+      field: 'selected',
+      type: SkyCellType.RowSelector,
+    },
+    {
+      field: 'name',
+      headerName: 'Name',
+    },
+    {
+      field: 'age',
+      headerName: 'Age',
+      type: SkyCellType.Number,
+      maxWidth: 60,
+    },
+    {
+      field: 'startDate',
+      headerName: 'Start Date',
+      type: SkyCellType.Date,
+      sort: 'asc',
+    },
+    {
+      field: 'endDate',
+      headerName: 'End Date',
```

Comment:
Sentence-style capitalization...
```suggestion
      headerName: 'End date',
```